app/components/RightSideBar/RightBarNav/RightBarNav.tsx

```diff
@@ -1,17 +1,88 @@
+'use client';
+
+import React, { useEffect, useState, useRef } from 'react';
 import styles from './RightBarNav.module.scss';
+import { useRouter, usePathname } from 'next/navigation';
 
 export function RightBarNav() {
+    const [isNotificationView, setIsNotificationView] = useState(false);
+    const [isDropdownOpen, setIsDropdownOpen] = useState(false);
+    const router = useRouter();
+    const pathname = usePathname();
+    const dropdownRef = useRef<HTMLDivElement>(null);
+
+    const toggleNotificationView = () => {
+        const newNotificationState = !isNotificationView;
+        setIsNotificationView(newNotificationState);
+
+        if (newNotificationState) {
+            router.push('/content-feed');
+        } else {
+            router.push('/');
+        }
+    };
+
+    const toggleDropdown = () => {
```

Comment:
external function

Reply:
This is so that when the user clicks on the notification button and then the page, the button does not remain colored.